doc/sphinx/proof-engine/vernacular-commands.rst

```diff
@@ -324,7 +324,7 @@ described elsewhere
 
          SearchPattern (?X1 + _ = _ + ?X1).
 
-.. cmd:: SearchRewrite @one_pattern {? {| inside | outside } {+ @qualid } }
+.. cmd:: SearchRewrite @one_pattern {? {| {| inside | outside } {+ @qualid } | in {+ @qualid } } }
```

Comment:
And also here:

![image](https://user-images.githubusercontent.com/1253341/140803850-655edfea-3da8-4d5b-a712-b3771c2a13ae.png)